### ISSUE TITLE
Automatically set user flag on basis of data imported from authentication provider

### ISSUE THREAD

**Issue by shkleinik** [JabbR/JabbR#505]
Automatically set user flag on basis of data imported from authentication provider

**Comment by adamralph** [comment]
But I'd like to have two flags - https://github.com/davidfowl/JabbR/issues/510

**Comment by shkleinik** [comment]
Why "but"? :) You can have 2 =)  1 defined automatically on your ip basis. And another one, which you treat suitable for you :)

**Comment by adamralph** [comment]
Fair enough - but I'd rather not have an automatic one based on my current IP due to privacy concerns (and security - I may not want a potential burglar to know that my house is empty because I'm on holiday on the Maldives). IMHO, this would have to be configurable, defaulting to disabled

In any case, IP based location is far from reliable. Perhaps the HTML5 geolocation stuff would be a better way of doing this, that way the allow/disallow is built right into the browser.

An automatic flag based on my openID details is another possibiliy, but I'm not sure what the 'country' an my openID details is supposed to signify. Location, nationality, something else?

**Comment by shkleinik** [comment]
Sounds meaningful

Country/location - more then enough :)
